server/lib/logger.js

```diff
@@ -0,0 +1,11 @@
+const MaskLogger = require('@financial-times/n-mask-logger').default;
+
+module.exports = new MaskLogger([
+	'email',
+	'password',
+	'contract_id',
+	'user_id',
```

Comment:
why do you want to mask user id? I think it could be pretty useful for debugging